### DISCUSSION TITLE
Basic Playmedia Request

### DISCUSSION THREAD

**Discussion by negmos** [music-assistant/support#1723] [Q&A] [answered]
I have just started using Music Assistant few days ago.
I have my server deployment running, with no homeassistant.
I would like to build my own telegram bot to send requests to the server to play specific playlists from my youtube music lists.
Any guidance on how to trigger a play_media request targeting specific player_id and playlist_id using python would be great.

**Comment by OzGav** [comment, marked as answer]
MA has service calls that you can use in scripts and automations.
